stories/Components/Navigationcomponents/Menu/Menu.jsx

```diff
@@ -3,9 +3,11 @@ import './menu.scss';
 import { MenuItems } from '../../../Atom/Navigation/Menu-items/Menu-Items';
 
 export const Menu = ({ data }) => (
-  <ul className="menu">
-    {data.map((item, index) => (
-      <li key={index} id={item.id}><MenuItems text={item.label} /></li>
-    ))}
-  </ul>
+  <div role="navigation" aria-label="menu">
```

Comment:
@harishkumar-srijan can't we use nav tag here

Reply:
done